{{cookiecutter.bot_name}}/app/db/sqlalchemy.py

```diff
@@ -0,0 +1,58 @@
+"""SQLAlchemy helpers."""
+
+from typing import Any
+
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+from app.settings.config import get_app_settings
+
+DATABASE_URL = get_app_settings().DATABASE_URL
+
+
+def make_url_async(url: str) -> str:
+    """Add +asyncpg to url scheme."""
+    return "postgresql+asyncpg" + url[url.find(":") :]  # noqa: WPS336
+
+
+def make_url_sync(url: str) -> str:
+    """Add +asyncpg to url scheme."""
+    return "postgresql" + url[url.find(":") :]  # noqa: WPS336
+
+
+Base = declarative_base()
+
+
+class AsyncDatabaseSession:
+    """Database session class."""
+
+    def __init__(self) -> None:
+        """Initialize."""
+        self._session = None
+        self._engine = None
+
+    def __getattr__(self, name: str) -> Any:
+        """Get session attrs by default."""
+        return getattr(self._session, name)
+
+    async def init(self) -> None:
+        """Async initialization."""
+        self._engine = create_async_engine(make_url_async(DATABASE_URL), echo=True)
+
+        self._session = sessionmaker(
+            self._engine, expire_on_commit=False, class_=AsyncSession
+        )()
+
+    async def create_all(self) -> None:
+        """Create database schema."""
+        assert self._engine is not None
+        async with self._engine.begin() as conn:
+            await conn.run_sync(Base.metadata.create_all)
```

Comment:
Why don't use migrations?

Reply:
This method is unused for now, and database schema creation is handled by migrations.
But this method may be useful as replacement of migrations to speed up the tests. Creating all schema's tables at once is much faster than one-by-one migrations applying.

Reply:
I think we should run database related tests separately (using `--db` flag for example). And we will get migration tests + fast tests runs. So for now, I suggest removing this code.

Reply:
moved to tests fixtures